src/vs/code/electron-main/backup.ts

```diff
@@ -0,0 +1,88 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
`IBackupService` manages all I/O with what is now the `IEnvironmentService.backupHome` directory.